web/geo/generate-aggregates.js

```diff
@@ -0,0 +1,51 @@
+const { union } = require('@turf/turf');
```

Comment:
On the overall process:

Your process may be better, in my head I was thinking of an approach where we'd

1. use 'reduce' to get a zoneToPolygon map
2. Iterate through map and union all those where length > 1

I think we'd be able to shave a few JS lines off but it might not be worth it.